docs/user/addons.rst

```diff
@@ -43,6 +43,32 @@ Individual configuration options for each addon are available in :guilabel:`Sett
 #. In the left bar, go to :guilabel:`Addons`.
 #. Configure each Addon individually.
 
+Integrating with Addons
+-----------------------
+
+Integrate with Search as you type
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To configure your site to use :doc:`Read the Docs search </server-side-search/index>` instead of the default search, add the following block of JavaScript:
```

Comment:
I think we should not be that direct here with "add the following block..." because we expect people to adapt/change this code. So, maybe something like "you can follow this as an example to adapt" or similar.